setup.py

```diff
@@ -16,6 +16,11 @@
 AUTHOR = "Antoine Martin"
 AUTHOR_EMAIL = "antoine@xpra.org"
 
+# List of configuration files
+# These must not be minified or compressed and must be moved to /etc/xpra
+# and symlinked to /usr/share/xpra/www if no custom installation directory
+# is set
+configuration_files = { "default-settings.txt" }
```

Comment:
You say this is a _List of configuration files_ but it is actually a set...
Then later on, you call it a `not_minify_dict`, but it isn't a dict.

Reply:
Let's settle with `Configuration files that must not be minified and compressed…` and `not_minify_files` then?

Reply:
OK!
I would use an array: `configuration_files = ["default-settings.txt"]` and `not_minify_files`.

Reply:
OK! Then let me push onve more ;)